### PR TITLE
Land "Make ceil,floor,round,trunc handle integers"

### PR DESCRIPTION
**Pull request by lezcano**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #85144

PR to land https://github.com/pytorch/pytorch/pull/78480, as Rohit does
not work in the PyTorch project anymore

### CODE REVIEW

**Comment by Lezcano** [comment]
@JackCaoG do you mind pointing https://github.com/pytorch/xla/pull/3913 to this one? We can coordinate the landing on Monday.

@ngimel could you OK this one?

**Review by ngimel — APPROVED** [review]

File: torch/_torch_docs.py
Comment (mruberry) [inline torch/_torch_docs.py:4170]: Also the convention in Python

**Comment by mruberry** [comment]
XLA and nvFuser failures look real and see suggested doc change inline.

I suggest xfailing those tests for now and citing two issues.

**Review by mruberry — APPROVED** [review]
Approving for velocity but tests will need to be addressed and docs could be tweaked (see comment).

Thanks @Lezcano!

**Comment by JackCaoG** [comment]
sg, I updated the xla pr, if all test passed I think we are good to merge.

**Comment by Lezcano** [comment]
@JackCaoG I added the pin. See what the CI has to say.

**Comment by JackCaoG** [comment]
@Lezcano Feel free to merge this one once all test is green, I will merge xla pr.

**Comment by Lezcano** [comment]
@janeyx99 I thought that `merge` by default would now run `-l`, so this should have been caught?

**Comment by janeyx99** [comment]
Oh no! Sorry for the revert--it definitely should have defaulted to -l and there seems to be a bug here! cc'ing @ZainRizvi to investigate.

**Comment by janeyx99** [comment]
@Lezcano can you try the pytorchbot merge command again when you're ready to merge something? I've merged https://github.com/pytorch/test-infra/pull/751 which I believe should fix the problem.

**Comment by Lezcano** [comment]
will do, thanks!

File: test/test_jit_fuser_te.py
Comment (Lezcano) [inline test/test_jit_fuser_te.py:1402]: Filed an issue https://github.com/pytorch/pytorch/issues/85342

**Comment by lezcano** [comment]
@ZainRizvi there are some tests in this PR that are taking forever to run (9h+). Do you know what's going on?

**Comment by ZainRizvi** [comment]
> @ZainRizvi there are some tests in this PR that are taking forever to run (9h+). Do you know what's going on?

Yeah, we're having some infra issues right now where we're severely under capacity on some machine types and it's causing tests to lag 😔

**Pull request closed without merging**
